pkg/operator/ceph/csi/controller.go

```diff
@@ -173,11 +173,15 @@ func (r *ReconcileCSI) reconcile(request reconcile.Request) (reconcile.Result, e
 		r.opConfig.Parameters = opConfig.Data
 	}
 
-	csiHostNetworkEnabled, err := strconv.ParseBool(k8sutil.GetValue(r.opConfig.Parameters, "CSI_ENABLE_HOST_NETWORK", "true"))
+	csiHostNetworkEnabled, err := strconv.ParseBool(k8sutil.GetValue(r.opConfig.Parameters, "CSI_ENABLE_HOST_NETWORK", useHostNetworkDefault))
 	if err != nil {
 		return reconcile.Result{}, errors.Wrap(err, "failed to parse value for 'CSI_ENABLE_HOST_NETWORK'")
 	}
 
+	if csiHostNetworkEnabled {
+		logger.Warning("CSI_ENABLE_HOST_NETWORK is set to true. This is not recommended when ceph cluster is reachable from Pod. Please consider using a pod networking.")
```

Comment:
Host networking can still be a common scenario for external clusters connecting to this scenario, so this warning doesn't seem needed.